### ISSUE TITLE
Virtual F13 key is not recognised

### ISSUE THREAD

**Issue by Avijith-Av** [microsoft/vscode#162503]
Steps to reproduce:
1. Set Windows key Left(or any other) to any key from F13 to F19 using Sharpkeys (tool to edit keycode using registry editor) or directly via registry editor. 
2. Vscode does not recognise said key. Other programs and windows itself does recognise the key with its newly assigned keycode

**Comment by Avijith-Av** [comment]
I passed this via vscode's internal report system and hence couldn't add some additional information, which i will add here.

1. When i use AutoHotKey to assign the key to itself (i.e. assign F13 to return F13), vscode suddenly recognises it. 
2. This was a recent change. I did not have this issue in the previous version of vscode. (present version: 1.17.2)

Please do let me know if i need to add some additional information.

**Comment by alexdima** [comment]
@Avijith-Av Does the remapping work when opening https://vscode.dev/ in Edge or Chrome?

**Comment by Avijith-Av** [comment]
> @Avijith-Av Does the remapping work when opening https://vscode.dev/ in Edge or Chrome?

No. the remapping does not work in https://vscode.dev/ . (tried in edge)
